core/consensus/authority/impl/authority_manager_impl.cpp

```diff
@@ -79,6 +79,13 @@ namespace kagome::authority {
     return outcome::success();
   }
 
+  std::optional<const primitives::BlockInfo> AuthorityManagerImpl::base() {
```

Comment:
Why optional?  If base is not initialized it is critical error, and application can not run.

Reply:
it is. fixed